src/stability_sdk/animation.py

```diff
@@ -809,6 +809,9 @@ def setup_animation(self, resume):
         if args.border == 'wrap' and args.animation_mode != '2D':
             args.border = 'reflect'
             logger.warning(f"Border 'wrap' is only supported in 2D mode, switching to '{args.border}'.")
+        if args.border == 'prefill' and args.animation_mode in ('2D', '3D warp') and args.inpaint_border == False:
```

Comment:
Should this be args.animation_mode != '3D render' instead of checking against 2D and 3D warp? Otherwise will hit same issue with Video Input potentially.

Reply:
Video Input mode doesn't apply any bordering, so the selected border is idle in this case